plugins/catalog-import/src/setupTests.ts

```diff
@@ -15,3 +15,6 @@
  */
 
 import '@testing-library/jest-dom';
+import fetch from 'cross-fetch';
+
+global.fetch = fetch;
```

Comment:
Oh sorry, didn't spot this one until now. Don't do this, just import `cross-fetch` in isomorphic code instead, and make sure that everything else is mocked.

Reply:
I moved this over to actual deployed code side. I don't know what you mean by `everything else is mocked` though.

Reply:
Anything else that might use fetch under the hood, like octokit

Reply:
Looking at this for a second time since moving this to actual dependencies is breaking the pipeline. It seems that templates are expecting `cross-fetch` to be a devDep and should be used as a global registration instead. Should we modify the templates to match the above?

Reply:
Taking a second look at this since moving `cross-fetch` to actual code side is making pipelines fail. The templates are currently wanting to put this in as a devDep and register it globally for tests instead of importing. Should we modify the templates to match the comment above?

Reply:
@Rugvip I've modified this to conform the templates.

Reply:
Yep, just noticed that as well. I just knew the `global` assignment was off but other than that I obviously didn't know what I was talking about x)